### PR TITLE
pallet_whitelist ready

### PR DESCRIPTION
**Pull request by wangminqi**

### CODE REVIEW

File: pallets/whitelist/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[package]
+authors = ['Litentry Dev']
+edition = '2021'
+homepage = 'https://litentry.com'
+name = 'pallet-whitelist'
+repository = 'https://github.com/litentry/litentry-pallets'
```
Comment (Kailai-Wang) [inline pallets/whitelist/Cargo.toml]: I think this URL is deprecated - let's just use parachain repo

File: pallets/whitelist/src/lib.rs
Comment (Kailai-Wang) [inline pallets/whitelist/src/lib.rs:1]: Do you want to use it as 1) a standalone pallet 2) a pallet that can be connected to other pallets (like `pallet-membership`)

If 1) => we don't need the EnsureOrigin
If 2) => we probably need multiple instance support, as there can be more pallets which use pallet-whitelist

File: pallets/whitelist/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+		}
+
+		/// Batch adding of new whitelists
+		#[pallet::call_index(1)]
+		#[pallet::weight(100_000)]
+		#[transactional]
+		pub fn batch_add_whitelists(origin: OriginFor<T>, vs: Vec<T::AccountId>) -> DispatchResult {
+			T::WhitelistManagerOrigin::ensure_origin(origin)?;
+			for v in vs {
+				ensure!(!Self::is_whitelist(&v), Error::<T>::WhitelistAlreadyExists);
```
Comment (Kailai-Wang) [inline pallets/whitelist/src/lib.rs]: I'm a bit undecided, should we continue or break? Same for the removal.

**Review by Kailai-Wang — COMMENTED** [review]
Looks good in general except several questions, please check below

File: pallets/whitelist/src/lib.rs
Comment (wangminqi) [reply pallets/whitelist/src/lib.rs:1]: Just adding a sample implementation for demosntration prupose.

File: pallets/whitelist/src/lib.rs (hunk 2 above)
Comment (wangminqi) [reply pallets/whitelist/src/lib.rs]: break. And I use [transactional] for further making it either all executed or none.

**Comment by wangminqi** [comment]
The pallet is now instantible

**Review by Kailai-Wang — APPROVED** [review]
Thanks, I'll take care of the rest:
- remove IMP-mock code
- remove litmus IMP and VCMP

**Pull request merged**